buildroot-external/overlay/base/etc/lighttpd/conf.d/proxy_normal.conf

```diff
@@ -27,22 +27,20 @@ $SERVER["socket"] == ":8181" {
   proxy.server =  ( "" => ("localhost" => ("host" => "127.0.0.1", "port" => 8183)))
   include "/var/etc/lighttpd_auth.conf"
 }
-$SERVER["socket"] == "[::]:8181" {
-  server.use-ipv6 = "enable"
```

Comment:
Is removing `server.use-ipv6` here really non-critical? AFAIR I also had to add that back when I added IPv6 support so that lighttpd really started to work correctly.

Reply:
lighttpd treats addresses that begin with `[` as `AF_INET6`.